crates/storage/db/src/tables/mod.rs

```diff
@@ -278,6 +280,16 @@ dupsort!(
     ( HashedStorage ) H256 | [H256] StorageEntry
 );
 
+table!(
+    /// Stores the current state of an [`Account`].
+    ( AccountsTrie ) H256 | Vec<u8>
```

Comment:
is the Vec<u8> supposed to be replaced by another type?

Reply:
Maybe. _cita-trie_ uses `Vec<u8>` as a bytes representation for encoded nodes, and we are putting it in the DB like that, but we don't know how it may affect performance. wdyt?